torch/_inductor/compile_worker/subproc_pool.py

```diff
@@ -145,11 +145,24 @@ def __init__(
             f"--write-fd={str(subproc_write_fd)}",
             f"--torch-key={torch_key_str}",
         ]
-        local = False
-        if config.worker_suppress_logging:
+        mast_job_id = os.environ.get("MAST_HPC_JOB_NAME", None)
+        global_rank = os.environ.get("ROLE_RANK", "0")
+        worker_log_path = os.environ.get("TORCHINDUCTOR_WORKER_LOGPATH", config.worker_log_path)
+        stdout_pipe = None
```

Comment:
FYI, stdout and stderr_pipe are now redundant and we can get rid of them and just use log_file directly.